Tests/test_file_gbr.py

```diff
@@ -0,0 +1,15 @@
+from helper import unittest, PillowTestCase
+
+from PIL import Image, GbrImagePlugin
+
+
+class TestFileGbr(PillowTestCase):
+
+    def test_invalid_file(self):
+        self.assertRaises(SyntaxError, lambda: GbrImagePlugin.GbrImageFile("Tests/images/flower.jpg"))
```

Comment:
I tried breaking between SyntaxError and lambda, or between lambda and GbrImagePlugin, but it still leaves a line > 80